### ISSUE TITLE
Game keeps crashing and noting instability on build 1.5.94.16

### ISSUE THREAD

**Issue by gannon345** [SolastaMods/SolastaUnfinishedBusiness#4041]
[Player.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/12711205/Player.log)
[Player-prev.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/12711211/Player-prev.log)

Errors appear to vary, haven't determined root cause yet, but had zero issues on previous builds.

**Comment by gannon345** [comment]
Sorry, forgot the character files.
[Characters.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/12711326/Characters.zip).

I have found the crash is repeatable in at least one spot though, occurs when I fire a crossbow at the boarded up window in dark castle at the start of the Aksha fight.
[Saves.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/12711335/Saves.zip)

I've attached the autosave from the start of the fight, and a quicksave from right outside the room prior.
